### PR TITLE
Advanced Penlight for CMO

### PR DESCRIPTION
**Pull request by solwashere**
# Document the changes in your pull request

Gives the CMO an advanced penlight, capable of cauterizing bleeding, forming the hologram that nobody uses, functioning as a light and sterilizing burn infections.

# Spriting

image
image

# Wiki Documentation

Add advanced penlight to medical tools.

# Changelog

:cl:  
rscadd: advanced penlight for the CMO, heals bleeding wounds, sterilizes burns better than the normal one, is a bit brighter

/:cl:

### CODE REVIEW

**Review by Moltijoe — CHANGES_REQUESTED** [review]
Please atomize this

**Comment by solwashere** [comment]
> Please atomize this

split the toolspeed bit into a separate PR?

**Comment by Hislyna** [comment]
I like the hologram....

**Review by cuackles — APPROVED** [review]

File: code/game/objects/items/devices/flashlight.dm [hunk 1]
```diff
@@ -201,6 +201,31 @@
...
+	if(!proximity_flag)
+		if(holo_cooldown > world.time)
+			to_chat(user, span_warning("[src] is not ready yet!"))
+			return
+		var/T = get_turf(target)
+		if(locate(/mob/living) in T)
+			new /obj/effect/temp_visual/medical_holosign(T,user) //produce a holographic glow
+			holo_cooldown = world.time + 10 SECONDS
+			return
+
```
Comment (Moltijoe) [inline code/game/objects/items/devices/flashlight.dm]: what's the point of this? regular penlights already have this afterattack

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/game/objects/items/devices/flashlight.dm (hunk 1 above)
Comment (solwashere) [reply code/game/objects/items/devices/flashlight.dm]: Fogor that it applied to all penlights and threw it in, didn't realize it'd cause a redundancy

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**